packages/deployments/subgraph/config/amarok-hub-v0.json

```diff
@@ -37,6 +37,11 @@
         "name": "BnbHubConnector",
         "address": "0x69009c6f567590d8B469dbF4C8808e8ee32b8a45",
         "startBlock": 17640454
+      },
+      {
+        "name": "LineaHubConnector",
+        "address": "0x076cD2B25cb1Ed7272d716bdeb4A8551CF606a3D",
+        "startBlock": 18468809
```

Comment:
I don't think we need `LineaHubConnector` address here if we don't have a network entity in subgraph.template.yaml. or maybe we are currently missing it in the template file???

Reply:
it is in the template file: https://github.com/connext/monorepo/blob/fad4783676bfa7124092b9d86dc691a89871234b/packages/deployments/subgraph/src/amarok-hub-v0/subgraph.template.yaml#L221-L247